runtime/src/main/java/com/flipkart/flux/resource/ClientElbResource.java

```diff
@@ -0,0 +1,175 @@
+package com.flipkart.flux.resource;
+
+import com.codahale.metrics.annotation.Timed;
+import com.flipkart.flux.api.ClientElbDefinition;
+import com.flipkart.flux.domain.ClientElb;
+import com.flipkart.flux.representation.ClientElbPersistenceService;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Named;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.UUID;
+
+/**
+ * API for CRUD operations on Client Cluster ELB details
+ * @author akif.khan
+ */
+@Singleton
+@Path("/api/clientelb")
```

Comment:
/api/client-elb is probably a better name.

Reply:
Changed endpoint to '/api/client-elb' here and at corresponding usage places.

Changes can be viewed in commit titled: Review Comments Resolution 2
Added a new commit because rebase was causing much more merge conflicts
(prone to error and subject to verification) and it was time consuming.